servicecontrol/servicecontrol-instances/remotes.md

```diff
@@ -116,6 +116,56 @@ class ServiceControl ServiceControlPrimary
 class ServiceControlAudit1,ServiceControlAudit2 ServiceControlRemote
 ```
 
+### Multi-transport deployments
+
+When using multiple transports the ServiceControl can be setup together with the [MessagingBridge](/nservicebus/bridge/) to allow the entire system to be managed via a single ServicePulse/ServiceInsight installation.
```

Comment:
Based on https://docs.particular.net/servicecontrol/servicecontrol-instances/remotes#overview-multi-region-deployments, it looks ServiceInsight can connect to the main instance without the use of the bridge, so is this only required for ServicePulse?

Reply:
yes that is correct but I figured that this is the way to go for both SC and SI so that wording is kind of correct?

Reply:
Yeah, that does make sense actually. Adding a single error instance pointing to two other error instances (as shown in that other diagram) would be an _alternate_ way do it, but just for SI, whereas as this is the way to do it for both SI and SP.